general/mem_manager.hpp

```diff
@@ -32,13 +32,15 @@ enum class MemoryType
    HOST_64,        ///< Host memory; aligned at 64 bytes
    HOST_DEBUG,     ///< Host memory; allocated from a "host-debug" pool
    HOST_UMPIRE,    ///< Host memory; using Umpire
+   HOST_PINNED,    ///< Host memory: pinned (page-locked)
```

Comment:
These and other changes in this PR should be mentioned in `CHANGELOG`

Reply:
I added some stuff to the `CHANGELOG`, is the level of detail okay? I didn't add anything on the class constructor changes that now let you specify temporary memory but I can add that if you want.

Reply:
Yes, thank you